docs/getting-started/offline-endpoint.asciidoc

```diff
@@ -0,0 +1,180 @@
+
+[source,sh]
+----
+cat > nginx.conf << EOF
+server {
+  root /app/static;
+  location / {}
+}
+EOF
+docker run -v "$PWD"/nginx.conf:/etc/nginx/conf.d/default.conf:ro -v "$PWD"/static:/app/static:ro -p 80:80 nginx
+----
+
+IMPORTANT: This example script is not appropriate for production environments. We recommend configuring the Nginx server to use http://nginx.org/en/docs/http/configuring_https_servers.html[TLS] according to your IT policies. Refer to https://docs.nginx.com/nginx/admin-guide/installing-nginx/[Nginx documentation] for more information on downloading and configuring Nginx. 
+====
+
+[discrete]
+._Example: Apache HTTPD_
+[%collapsible]
+====
+This example script starts an Apache httpd Docker image and configures it as a file server:
+
+[source,sh]
+----
+docker run -p 80:80 -v "$PWD":/usr/local/apache2/htdocs/ httpd
+----
+
+IMPORTANT: This example script is not appropriate for production environments. We recommend configuring httpd to use https://httpd.apache.org/docs/trunk/ssl/ssl_howto.html[TLS] according to your IT policies. Refer to https://httpd.apache.org[Apache documentation] for more information on downloading and configuring Apache httpd.
+====
+
+[discrete]
+=== Step 2: Configure {elastic-endpoint}
+
+Set the `advanced.artifacts.global.base_url` advanced setting for each <<configure-endpoint-integration-policy,{elastic-defend} integration policy>> that needs to use the mirror. Note that there's a separate setting for each operating system:
+
+* `linux.advanced.artifacts.global.base_url`
+* `mac.advanced.artifacts.global.base_url`
+* `windows.advanced.artifacts.global.base_url`
+
+[role="screenshot"]
+image::images/offline-adv-settings.png[Integration policy advanced settings]
+
+[discrete]
+=== Step 3: Manually copy artifact updates
+
+Download the most recent artifact files from the Elastic global artifact server, then copy those files to the server instance you created in step 1.
+
+Below is an example script that downloads all the global artifact updates. There are different artifact files for each version of {elastic-endpoint}. Change the value of the `ENDPOINT_VERSION` variable in the example script to match the deployed version of {elastic-endpoint}.
+
+[source,sh,subs="attributes"]
+----
+export ENDPOINT_VERSION={version} && wget -P downloads/endpoint/manifest https://artifacts.security.elastic.co/downloads/endpoint/manifest/artifacts-$ENDPOINT_VERSION.zip && zcat -q downloads/endpoint/manifest/artifacts-$ENDPOINT_VERSION.zip | jq -r --arg root "https://artifacts.security.elastic.co" '.artifacts | to_entries[] | $root + .value.relative_url' | xargs wget -P downloads/endpoint -c
+----
+
+This command will download files and directory structure that should be directly copied to the file server.
+
+Elastic releases updates on a continuous basis as detection engines are improved. We recommend updating air-gapped environments monthly to stay current with artifact updates.
```

Comment:
I put some thought on "monthly". I think in general monthly is a good cadence for most. Very regular updates are most advantageous for at-risk, internet connected devices, not air-gapped systems. That said, maybe a slight benefit for going weekly. I would be good leaving this as is or adding a slight qualifier for "at least least monthly".

Reply:
I'll defer to the SMEs on this detail

@stevewritescode @hcarty et al.

Reply:
I'm cool revising this from `monthly` to `at least monthly`